app/views/shared/_sponsors.html.erb

```diff
@@ -0,0 +1,61 @@
+ <div class="jumbotron" align="middle">
+  <h1>Proudly supported by</h1>
+
+  <div class="container marketing">
```

Comment:
I'm not sure what the marketing class does, I inspected the page and couldn't find anything. Does this need to be here?